File: products/session_vault_mcp.py
#!/usr/bin/env python3
"""Deprecated entrypoint — use products/mcp_cf_bots_mcp.py and MCP name mcp-cf-bots."""

from __future__ import annotations

import asyncio
import sys

if __name__ == "__main__":
    print(
        "session_vault_mcp.py is deprecated; use mcp_cf_bots_mcp.py (MCP id: mcp-cf-bots)",
        file=sys.stderr,
    )
    from mcp_cf_bots_mcp import _main

    asyncio.run(_main())

File: products/session_vault_mcp.ts
#!/usr/bin/env bun
/** @deprecated Use products/mcp_cf_bots_mcp.ts and MCP id mcp-cf-bots */

if (import.meta.main) {
  console.error("session_vault_mcp.ts is deprecated; use mcp_cf_bots_mcp.ts");
  const path = new URL("./mcp_cf_bots_mcp.ts", import.meta.url).pathname;
  const proc = Bun.spawn(["bun", path, ...process.argv.slice(2)], {
    stdin: "inherit",
    stdout: "inherit",
    stderr: "inherit",
  });
  process.exit(await proc.exited);
}
